### PR TITLE
Update license year

### PR DESCRIPTION
**Pull request by m1guelpf**
TO DO:

- [x] Wait to the next year (100% done, depends on timezone)
- [ ] Merge!

:octocat:

### CODE REVIEW

File: LICENSE.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2017 Airbnb
+Copyright (c) 2018 Airbnb
```
Comment (ljharb) [inline LICENSE.md]: Technically these years don’t need to be updated at all, legally; I’d prefer to just set this back to the original year the guide was created and leave it be forever.

Comment (m1guelpf) [reply LICENSE.md]: What about {year}-present?

File: LICENSE.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2017 Airbnb
+Copyright (c) 2017-present Airbnb
```
Comment (ljharb) [inline LICENSE.md]: The copyright didn't start in 2017; this should just say "2012". The " - present" is always implied and thus is redundant.

**Comment by m1guelpf** [comment]
@ljharb Changed to original date

**Pull request merged**